java/code/src/com/suse/manager/maintenance/MaintenanceManager.java

```diff
@@ -580,18 +583,24 @@ else if (operation.equals("skipNext")) {
             date = nextWindow.get().getFromMilliseconds();
         }
 
-        Map<String, Long> activeRange = getActiveRange(date);
+        Map<String, Long> activeRange = getActiveRange(user, date, startOfWeek);
         Long start = activeRange.get("start");
         Long end = activeRange.get("end");
 
         return icalUtils.getCalendarEvents(calendar, eventName, start, end);
     }
 
-    private Map<String, Long> getActiveRange(Long date) {
+    private Map<String, Long> getActiveRange(User user, Long date, Long startOfWeek) {
```

Comment:
The parameter `user` does not seem to be used.

Reply:
Yeah it was removed in a later commit.